data/portfolioinfo2.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "companyName": "Hilton",
+    "companyLink": "http://stayhgv.com/",
+    "experience": "<p>Candy canes donut lemon drops gummies caramels. Candy canes cake cupcake toffee jelly beans jelly beans powder. Pastry dragée sweet roll jujubes pastry muffin lemon drops toffee. Cotton candy candy oat cake dragée candy canes jelly.</p>"
+  },
+  {
+    "companyName": "Marriott",
+    "companyLink": "http://www.marriottvacationclub.com/",
+    "experience": "<p>Sweet soufflé bonbon pie chocolate. Liquorice chupa chups cake chupa chups donut. Candy sesame snaps carrot cake carrot cake fruitcake jujubes powder. Tart jelly beans chocolate bar tart pie jelly-o jelly beans.</p>"
+  },
+  {
+    "companyName": "KDDI Evolva",
+    "companyLink": "http://www.k-evolva.com/eng/",
+    "experience": "<p>Ice cream danish toffee sesame snaps chocolate gingerbread oat cake sesame snaps. Fruitcake sesame snaps pudding gummi bears soufflé. Sweet roll jujubes cake lollipop croissant pudding. Croissant jujubes dragée marzipan brownie.</p>"
+  },
```

Comment:
is this working? make sure to remove the last comma in an array for json files or it can break everything